apps/framework-cli/src/framework/core/infrastructure/table.rs

```diff
@@ -172,8 +172,8 @@ pub struct EnumMemberMetadata {
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq)]
 #[serde(untagged)]
 pub enum EnumValueMetadata {
-    /// Integer value for numeric enums
-    Int(u8),
+    /// Integer value for numeric enums (supports Enum8: -128 to 127, Enum16: -32768 to 32767)
+    Int(i16),
```

Comment:
This kinda sketches me out but feels safe since u8 is a subset of i16 anyways. Though this kinda forces us to follow the clickhouse thing of supporting negative enum ids.